data_model/src/query/mod.rs

```diff
@@ -40,6 +41,28 @@ pub mod cursor;
 pub mod pagination;
 pub mod sorting;
 
+const FETCH_SIZE: &str = "fetch_size";
+
+/// Default value for `fetch_size` parameter in queries.
+// SAFETY: `100` is greater than `0`
+#[allow(unsafe_code)]
+pub const DEFAULT_FETCH_SIZE: NonZeroU32 = unsafe { NonZeroU32::new_unchecked(100) };
```

Comment:
```suggestion
pub const DEFAULT_FETCH_SIZE: NonZeroU32 = unsafe { NonZeroU32::new_unchecked(10) };
```

in JDBC most drivers use `10`